src/Promitor.Integrations.Sinks.Prometheus/Collectors/PrometheusSystemMetricsSink.cs

```diff
@@ -36,5 +36,23 @@ public Task WriteGaugeMeasurementAsync(string name, string description, double v
 
             return Task.CompletedTask;
         }
+        
+        /// <summary>
+        ///     Records measurement for a histogram instrument 
+        /// </summary>
+        /// <param name="name">Name of the metric</param>
+        /// <param name="description">Description of the metric</param>
+        /// <param name="value">New measured value</param>
+        /// <param name="labels">Labels that are applicable for this measurement</param>
+        /// <param name="includeTimestamp">Indication whether or not a timestamp should be reported</param>
+        public Task WriteHistogramMeasurementAsync(string name, string description, double value, Dictionary<string, string> labels, bool includeTimestamp)
+        {
+            var orderedLabels = labels.OrderByDescending(kvp => kvp.Key).ToDictionary(kvp => kvp.Key, kvp => kvp.Value);
```

Comment:
What if labels are null?

Reply:
Added a null check for this histogram and the gauge function above